packages/orm/src/EntityManager.ts

```diff
@@ -180,8 +180,8 @@ export class EntityManager<C = {}> {
   }
 
   /** Looks up `id` in the list of already-loaded entities. */
-  getEntity<T extends Entity>(id: IdOf<T>): T | undefined {
-    return this._entityIndex.get(id) as T | undefined;
+  getEntity<T extends Entity>(tableName: string, id: IdOf<T>): T | undefined {
```

Comment:
Our codebase uses this method a lot, and just in general I think/hope that eventually most Joist apps will use tagged ids, and so should be allowed to use this method.

Can we put it back, and then add an `assertTaggedId`-style check that will fail at runtime if its called w/an untagged id?